### ISSUE TITLE
Screen readers: Cannot customize keyboard shortcuts in MS4

### ISSUE THREAD

**Issue by msviolafangirl** [musescore/MuseScore#17923]
### Issue type

UI bug

### Bug description

The keyboard shortcut entry window doesn't really work properly, at least not with Windows and NVDA.

### Steps to reproduce

1. Go to preferences.
2. Go to shortcuts.
3. Choose  shortcut to edit.
4. Press enter.
5. Press a combination of keys. My screen reader gives me no feedback when doing this.
6. Press escape.
7. See if any changes have been made. When I go through this process, it seems that the shortcut I entered not saved.

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/91112676/ad4aa207-3b4d-4800-ac04-ac53a099ec8d

### MuseScore Version

4.0.2

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 10

**Comment by MarcSabatella** [comment]
Thanks, this is already reported as https://github.com/musescore/MuseScore/issues/16157

**Comment by msviolafangirl** [comment]
Thanks Marc for letting me know.
